includes/class-wp-job-manager-shortcodes.php

```diff
@@ -625,6 +627,9 @@ public function output_jobs( $atts ) {
 		if ( ! is_null( $atts['remote_position'] ) ) {
 			$atts['remote_position'] = ( is_bool( $atts['remote_position'] ) && $atts['remote_position'] ) || in_array( $atts['remote_position'], [ 1, '1', 'true', 'yes' ], true );
 		}
+		if ( ! is_null( $atts['featured_first'] ) ) {
+			$atts['featured_first'] = ( is_bool( $atts['featured_first'] ) && $atts['featured_first'] ) || in_array( $atts['featured_first'], [ 1, '1', 'true', 'yes' ], true );
+		}
```

Comment:
```suggestion
```
$this->string_to_bool is already called on $atts['featured_first'], which does exactly that

Reply:
Removed in d716a58bdd6dcb2742017797582c671869725f4b